notification-portlet-webapp/src/main/webapp/less/accordion.less

```diff
@@ -93,10 +93,13 @@
                     margin: 0;
 
                     li {
-                        list-style: none !important;
-                        line-height: 18px;
-                        font-size: 16px;
-                        margin: 10px 0;
+                        list-style-image: url("../images/chevron-right.png");
+                        font-size: 20px;
```

Comment:
This change makes the text quite a bit bigger.  To me it seems out of proportion with the category names in the clickable accordion segments.